src/test/Business/MetaQuestionController.test.js

```diff
@@ -75,7 +64,7 @@ describe('Happy-Path MetaQuestionController tests', () => {
     });
 
     beforeEach(async () => {
-        mqController = await new MetaQuestionController(mqRepo, new TaskControllerMock(), new UserControllerMock());
+        mqController = await new MetaQuestionController(mqRepo);
```

Comment:
I am not sure if we should create mock for our repos ( I thinking about testing the Exam controller)

Should we leave it this way and check integration, or create a mock to really do a unit test?

Reply:
I removed the mocks! I removed the coupling of `MQController` with `TaskController` and `UserController` entirely!